<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Главное меню</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
   background-color: #13171c;
   color: #f5f5f5;
   font-family:  Rubik;
}

QLineEdit {
	border: 2px solid #cfcfcf;
	border-radius: 4px;
}

QPushButton {
    background-color: #1d2024;
	border: none;
 	border-collapse: separate;
	border-radius: 5px;
}

QPushButton::disabled {
    background-color: #202024;
}

QPushButton::hover {
    background-color: #222a33;
}

QPushButton::pressed {
    background-color: #1d2329;
}

QListWidget {
	border: 2px solid #cfcfcf;
	border-radius: 7px;
}</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="mainWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minimizeButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>_</string>
    </property>
   </widget>
   <widget class="QWidget" name="navigationWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>61</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
   background-color: #586675;
}

QPushButton::hover {
   background-color: #505d6b;
}

QPushButton::pressed {
   background-color: #44515e;
}</string>
    </property>
    <widget class="QPushButton" name="profilesButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>P</string>
     </property>
    </widget>
    <widget class="QPushButton" name="settingsButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>S</string>
     </property>
    </widget>
    <widget class="QLabel" name="title">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rubik</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ffffff;</string>
     </property>
     <property name="text">
      <string>AB</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="adButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>400</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>GitHub</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="settingsWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>541</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="settingsTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rubik</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Настройки</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="trayCheckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>511</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rubik</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Прятать приложение в трей при нажатии кнопки &quot;Закрыть&quot;</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="autorunCheckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>511</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rubik</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Запускать приложение вместе с системой (автозапуск)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="nameCheckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>511</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rubik</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Использовать бинды не только в RAGE Multiplayer</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="profilesWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>541</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QPushButton" name="createProfileButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>80</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rubik</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>N</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editProfileButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>150</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rubik</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>E</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteProfileButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>220</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rubik</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>D</string>
     </property>
    </widget>
    <widget class="QLabel" name="profilesTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rubik</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Профили</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>431</width>
       <height>321</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rubik</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ваши профили:</string>
     </property>
    </widget>
    <zorder>listWidget</zorder>
    <zorder>createProfileButton</zorder>
    <zorder>editProfileButton</zorder>
    <zorder>deleteProfileButton</zorder>
    <zorder>profilesTitle</zorder>
    <zorder>label</zorder>
   </widget>
   <zorder>settingsWidget</zorder>
   <zorder>profilesWidget</zorder>
   <zorder>minimizeButton</zorder>
   <zorder>closeButton</zorder>
   <zorder>navigationWidget</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
